### PR TITLE
[DE] Update `osu!api`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!api/de.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
----
-
 # osu!api
 
 osu! hat eine öffentliche API, die Entwickler nutzen können, um die osu! Plattform zu unterstützen und zu erweitern. Früher war sie eher privat und konnte nur auf Anfrage genutzt werden, mittlerweile kann sie allerdings jeder nutzen der über einen osu! Account verfügt.
 
-Du kannst eine Dokumentation von der API auf der [Wikiseite der osu!api](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungsseite](https://osu.ppy.sh/p/api) besorgen.
+Du kannst eine Dokumentation von der API auf der [Wikiseite der osu!api](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungsseite](https://osu.ppy.sh/p/api) besorgen. Denke aber daran, dass die Dokumentation der osu!api eventuell nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird.
```
Comment (respektive) [inline wiki/osu!api/de.md:5]:
```suggestion
Du kannst eine Dokumentation von der API auf der [Wikiseite der osu!api](https://github.com/ppy/osu-api/wiki) finden. Die aktuelle Version der API nutzt private Keys für die Authentifizierung der Anfragen. Du kannst dir einen API Key auf der [Beantragungsseite](https://osu.ppy.sh/p/api) der osu!-Website besorgen. Denke aber daran, dass die Dokumentation der osu!api eventuell nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird.
```

File: wiki/osu!api/de.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+
+---
+
+Version 2 der osu!api ist gerade in Arbeit und zu finden im [osu-web](https://github.com/ppy/osu-web). Die Dokumentation (keine Garantie, dass diese vollständig ist) findet sich auf ihrer [Referenzseite](https://docs.ppy.sh).
```
Comment (respektive) [inline wiki/osu!api/de.md:9]:
```suggestion
Version 2 der osu!api ist in Arbeit unter [osu-web](https://github.com/ppy/osu-web), und hat das Ziel, viel robuster zu sein als das Original. Die Dokumentation (keine Garantie, dass diese vollständig ist) findet sich auf ihrer [Referenzseite](https://docs.ppy.sh).
```

File: wiki/osu!api/de.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
 
-Denke aber daran, dass die Dokumentation der osu!api eventuell nicht vollständig oder umfangreich ist, und die API nicht aktiv entwickelt wird. Eine neue und bessere API ist gerade in Arbeit und zu finden im [osu-web](https://github.com/ppy/osu-web).
+Die Authentifizierung für die neue API folgt dem [OAuth-2-Protokoll](https://oauth.net/2/). Clients können auf der [Einstellungsseite](https://osu.ppy.sh/home/account/edit) verwaltet werden.
```
Comment (respektive) [inline wiki/osu!api/de.md:11]:
```suggestion
Die Authentifizierung für die neue API folgt dem [OAuth-2-Protokoll](https://oauth.net/2/). Clients können auf der [Einstellungen Seite](https://osu.ppy.sh/home/account/edit) verwaltet werden.
```
page itself is called `Einstellungen` in german so might consider using this instead but either is fine imo

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/osu!api/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/osu!api/de.md:11]: How about `auf der Seite "Einstellungen"`?

Comment (respektive) [reply wiki/osu!api/de.md:11]: that sounds even better

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**